packages/bpk-component-button/src/BpkButtonBase.module.scss

```diff
@@ -47,3 +47,46 @@
     fill: currentcolor;
   }
 }
+
+.featured {
+  composes: bpk-button;
+  @include buttons.bpk-button--featured;
+}
+
+.destructive {
+  composes: bpk-button;
```

Comment:
CSS module feature: https://github.com/css-modules/css-modules/blob/master/docs/composition.md

style linter doesn't recognise it though